client/tests/phpunit/Pact/SiriusApiGatewayClientTest.php

```diff
@@ -0,0 +1,244 @@
+<?php declare(strict_types=1);
+
+namespace AppBundle\Service;
+
+
+use AppBundle\Service\AWS\RequestSigner;
+use AppBundle\Service\Client\Sirius\SiriusApiGatewayClient;
+use DateTime;
+use DigidepsTests\Helpers\DocumentHelpers;
+use DigidepsTests\Helpers\MultipartPactRequest;
+use DigidepsTests\Helpers\SiriusHelpers;
+use GuzzleHttp\Client as GuzzleClient;
+use GuzzleHttp\Psr7\Request;
+use PhpPact\Consumer\InteractionBuilder;
+use PhpPact\Consumer\Matcher\Matcher;
+use PhpPact\Consumer\Model\ConsumerRequest;
+use PhpPact\Consumer\Model\ProviderResponse;
+use PhpPact\Standalone\MockService\MockServerEnvConfig;
+use Prophecy\Argument;
+use Prophecy\Prophecy\ObjectProphecy;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+
+class SiriusDocumentsContractTest extends KernelTestCase
+{
+    /** @var string */
+    private $caseRef;
+
+    /**  @var GuzzleClient */
+    private $client;
+
+    /** @var RequestSigner&ObjectProphecy */
+    private $signer;
+
+    /** @var string */
+    private $baseUrl;
+
+    /** @var object|null  */
+    private $serializer;
+
+    /**  @var SiriusApiGatewayClient */
+    private $sut;
+
+    /** @var string */
+    private $reportPdfUuid;
+
+    /** @var string */
+    private $expectedSupportingDocumentUuid;
+
+    public function setUp(): void
+    {
+        $this->caseRef = '1234567T';
+        $this->reportPdfUuid = '33ea0382-cfc9-4776-9036-667eeb68fa4b';
+        $this->expectedSupportingDocumentUuid = '9c0cb55e-718d-4ffb-9599-f3164e12dbdb';
+        $this->client = new GuzzleClient();
+        $this->signer = self::prophesize(RequestSigner::class);
+        $this->baseUrl = getenv('PACT_MOCK_SERVER_HOST');
+        $this->serializer = (self::bootKernel(['debug' => false]))->getContainer()->get('serializer');
+        $this->sut = new SiriusApiGatewayClient($this->client, $this->signer->reveal(), 'http://' . $this->baseUrl, $this->serializer);
```

Comment:
Can simplify this a bit since there's no need to set them as properties (as they're never reused):
```suggestion
        $client = new GuzzleClient();
        $this->signer = self::prophesize(RequestSigner::class);
        $baseUrl = getenv('PACT_MOCK_SERVER_HOST');
        $serializer = (self::bootKernel(['debug' => false]))->getContainer()->get('serializer');
        $this->sut = new SiriusApiGatewayClient($client, $this->signer->reveal(), 'http://' . $baseUrl, $serializer);
```